Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM ghcr.io/lolopinto/ent:0.0.33-nodejs-17-slim
+FROM ghcr.io/lolopinto/ent:0.1.0-alpha.5-nodejs-17-dev
```

Comment:
can you update to `ghcr.io/lolopinto/ent:0.1.0-alpha.30-nodejs-17-slim`

that's the latest